Dockerfile

```diff
@@ -1,62 +1,37 @@
-# system libraries of general use
-RUN apt-get update && apt-get install -y \
-    sudo \
-    pandoc \
-    pandoc-citeproc \
-    libcurl4-gnutls-dev \
-    libcairo2-dev \
-    libxt-dev \
-    libssl-dev \
-    libssh2-1-dev \
-    libssl1.0 \
+RUN apt-get update && apt-get install -y --no-install-recommends \
     gdal-bin \
-    libgdal-dev \
-    libproj-dev \
-    libgeos-dev \
-    libudunits2-dev \
-	libgit2-dev
-	
-# Dependencies for rgdal and rgeos
-RUN  apt-get update && apt-get install -y software-properties-common && \
-     apt install gdal-bin
-	 
-# Dependencies for devtools
-RUN R -e "install.packages(c('gert', 'usethis'), repos='https://cloud.r-project.org/')"	 
-
-# install imports of reporting-grofwild app that are not on cloud
-RUN R -e "install.packages(c('shiny', 'sp', 'plyr', 'devtools', 'methods', 'reshape2', 'mgcv', 'rgdal', 'rgeos', 'raster', 'stringr'), repos = 'https://cloud.r-project.org/')"
-RUN R -e "devtools::install_version('plotly', version = '4.9.2.1', repos = 'http://cran.us.r-project.org')"
-RUN R -e "devtools::install_github('inbo/INBOtheme')"
-RUN R -e "install.packages('https://cran.r-project.org/src/contrib/Archive/DT/DT_0.12.tar.gz', repos = NULL, type = 'source')"
-RUN R -e "install.packages(c('flexdashboard'), repos='https://cloud.r-project.org/')"
-
-# install depends of reporting-grofwild app
-RUN R -e "install.packages(c('maptools'), repos='https://cloud.r-project.org/')"
-
-# install suggests of reporting-grofwild app
-RUN R -e "install.packages(c('leaflet', 'mapview'), repos='https://cloud.r-project.org/')"
+    libproj15 \
+    libgeos-3.8.0 libgeos-c1v5  \
+    curl \ 
+    ca-certificates \
+    pandoc \
+    libudunits2-0 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Use the remotes package instead of devtools as it is mutch lighter
+RUN R -q -e "install.packages('remotes')"
+
+RUN R -q -e "remotes::install_cran(c('shiny', 'sp', 'plyr', 'reshape2', 'mgcv', 'rgdal', 'rgeos', 'raster', 'stringr', 'maptools', 'leaflet', 'mapview', 'flexdashboard'))"
+RUN R -q -e "remotes::install_version('plotly', version = '4.9.2.1')"
+RUN R -q -e "remotes::install_version('DT', version = '0.12')"
+RUN R -q -e "remotes::install_github('inbo/INBOtheme')"
 
 # to prevent bobbing with shinycssloaders
-RUN R -e "remotes::install_github('daattali/shinycssloaders')"
+RUN R -q -e "remotes::install_github('daattali/shinycssloaders')"
 
 # For downloading the maps
 # Attention: do not install phantomjs directly, will not work then!
-RUN R -e "webshot::install_phantomjs()"
-
- 
-# copy the app to the image by installing package (need latest version!!)
-ENV latestApp reporting-grofwild.tar.gz
-COPY $latestApp /root/
-RUN R CMD INSTALL /root/$latestApp
-RUN rm /root/$latestApp
+RUN R -q -e "webshot::install_phantomjs()"
 
+# Install the package without the source files ending up in the Docker image
+RUN --mount=target=/tmp/package,source=reporting-grofwild R -q -e "remotes::install_local('/tmp/package', dependencies=FALSE)"
```

Comment:
when trying to build (`docker build -t openanalytics/r-base .`)using docker desktop I get the following error:
`Error response from daemon: Dockerfile parse error line 31: Unknown flag: mount`

Reply:
Can you try building the image with BuildKit enabled? You can enable it in the settings of Docker Engine in Docker Desktop (https://docs.docker.com/develop/develop-images/build_enhancements/#to-enable-buildkit-builds) or set a environment variable:

``bat
set DOCKER_BUILDKIT=1
docker build -t grofwild .
``

Reply:
I've enable the BuildKit, this is the content of the config file:
```
{
  "registry-mirrors": [],
  "insecure-registries": [],
  "debug": true,
  "experimental": false,
  "features": {
    "buildkit": true
  }
}
```
The initial error disappeared but a new, but related, error arose:
```
[+] Building 0.2s (2/2) FINISHED
 => [internal] load .dockerignore                                                                                                                                                                                                       0.1s
 => => transferring context: 2B                                                                                                                                                                                                         0.0s
 => [internal] load build definition from Dockerfile                                                                                                                                                                                    0.1s
 => => transferring dockerfile: 1.44kB                                                                                                                                                                                                  0.0s
failed to solve with frontend dockerfile.v0: failed to create LLB definition: Dockerfile parse error line 31: Unknown flag: mount
```

Reply:
PS I'm a docker noob

Reply:
I've updated my docker, this fixed this issue